src/network/networkd-radv.c

```diff
@@ -782,6 +864,35 @@ void network_drop_invalid_route_prefixes(Network *network) {
                         route_prefix_free(p);
 }
 
+static int pref64_prefix_section_verify(pref64Prefix *p) {
+        if (section_is_invalid(p->section))
+                return -EINVAL;
+
+        if (p->prefixlen > 128)
+                return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
+                                         "%s: Invalid prefix length %u is specified in [IPv6pref64Prefix] section. "
+                                         "Valid range is 0…128. Ignoring [IPv6pref64Prefix] section from line %u.",
```

Comment:
```suggestion
                                         "Valid lengths are \32, \40, \48, \56, \64 and \96. Ignoring [IPv6pref64Prefix] section from line %u.",
```

Not all prefix lengths are representable in the PLC, so this should be clarified further.